Tools/autotest/param_metadata/rstemit.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, *args, **kwargs):
         self.t = ''
 
     def escape(self, s):
-        ret = re.sub(self.rstescape, "\\\\\g<1>", s)
+        ret = re.sub(self.rstescape, "\\\\\g<1>", s)    # noqa
```

Comment:
This looks wrong.  I think you want
```suggestion
        ret = re.sub(self.rstescape, r"\\\\\g<1>", s)
```

Reply:
I was trying to fix the flake8 message from this line and python didn't like me fixing what flake8 was complaining about. So much for blindly following what the computer says. Output is unchanged with the raw string and fixing the escape count. Looks better too.